includes/admin/import/class-batch-import-payments.php

```diff
@@ -447,39 +454,46 @@ private function set_customer( $row ) {
 			}
 
 			// Make sure we found a customer. Create one if not.
-			if( empty( $customer->id ) ) {
+			if ( empty( $customer->id ) ) {
 
 				if ( ! $customer instanceof EDD_Customer ) {
-					$customer = new EDD_Customer;
+					$customer = new EDD_Customer();
 				}
+			}
+
+			$name = '';
```

Comment:
This isn't needed because it's always set no matter what in the following if/else statement.